lib/flag-filter.js

```diff
@@ -184,11 +184,15 @@ FlagFilter.prototype._transform = function (vm, enc, cb) {
 		vm.ptrname = vm.tags[DOCKER_PREFIX + INST_PTR_TAG];
 	}
 
-	/* And enable VMs owned by admin that have a service attached. */
-	if (vm.owner.login === 'admin' && vm.services.length > 0) {
+	/* 
+	 * And enable VMs owned by admin or poseidon that have a service 
+	 * attached. 
+	 */
+	if ((vm.owner.login === 'admin' || vm.owner.login === 'poseidon')
+		&& vm.services.length > 0) {
```

Comment:
Style: this needs to be a half-indent (4 spaces) past the tabstop of the if() line. (so, this line should start with one hard tab and then 4 spaces)